packages/web/playwright.config.ts

```diff
@@ -0,0 +1,110 @@
+import { defineConfig, devices } from '@playwright/test'
+import fs from 'fs'
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+const authFileExists = fs.existsSync('playwright/.auth/user.json')
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './e2e',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 1 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: process.env.CI
+    ? [
+        ['junit', { outputFile: 'report.xml', outputDir: 'playwright-report' }],
+        ['html', { open: 'never' }],
+        ['line'],
+        ['blob']
+      ]
+    : 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: 'http://localhost:3001',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+
+    /* Set the default timeout for actions like "click" */
+    actionTimeout: 10 * 1000
+  },
+
+  /* Total timeout for individual tests */
+  timeout: 5 * 60 * 1000,
+
+  /* Timeout for each assertion */
+  expect: {
+    timeout: 10 * 1000
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'setup',
+      testMatch: /.*\.setup.ts/
+    },
+    {
+      name: 'chromium',
+      dependencies: authFileExists ? undefined : ['setup'],
+      testIgnore: /.*\.setup.ts/,
+      use: {
+        ...devices['Desktop Chrome'],
+        storageState: 'playwright/.auth/user.json'
+      }
+    }
+
+    // {
+    //   name: 'firefox',
+    //   use: { ...devices['Desktop Firefox'] },
+    // },
+
+    // {
+    //   name: 'webkit',
+    //   use: { ...devices['Desktop Safari'] },
+    // },
+
+    /* Test against mobile viewports. */
+    // {
+    //   name: 'Mobile Chrome',
```

Comment:
no action for now since we should def limit ourselves to getting everything 100% on chromium; but I do wonder if/when we should enable more browsers. These mobile ones in particular seem like they might be handy